Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -141,6 +141,9 @@ AtomPDBResidueInfo *AtomGetPDBResidueInfo(Atom *atom) {
   return (AtomPDBResidueInfo *)res;
 }
 
+struct MDLDummy {};
```

Comment:
It was an experiment, I'm happy I know how to do this now.

In previous wrappings I hated the fact that namespaces weren't directly recapitulated in the python wrappers, I think namespaces are a good thing ;)